aspnetcore/tutorials/publish-to-azure-api-management-using-vs.md

```diff
@@ -4,192 +4,147 @@ author: codemillmatt
 
 By [Matt Soucoup](https://twitter.com/codemillmatt)
 
+In this tutorial you'll learn how to create an ASP.NET Core web API project using Visual Studio, ensure it has OpenAPI support, and then publish the web API to both Azure App Service and Azure API Management.
+
 ## Set up
 
-- Open a [free Azure account](https://azure.microsoft.com/free/dotnet/) if you don't have one.
-- [Create a new Azure API Management instance](/azure/api-management/get-started-create-service-instance) if you haven't already.
-- [Create a web API app project](xref:tutorials/first-web-api#create-a-web-project).
+To complete the tutorial you'll need an Azure account.
 
-## Configure the app
+* Open a [free Azure account](https://azure.microsoft.com/free/dotnet/) if you don't have one.
 
-Adding Swagger definitions to the ASP.NET Core web API allows Azure API Management to read the app's API definitions. Complete the following steps.
+## Create an ASP.NET Core web API
 
-### Add Swagger
+Visual Studio allows you to easily create a new ASP.NET Core web API project from a template. Follow these directions to create a new ASP.NET Core web API project:
 
-1. Add the [Swashbuckle.AspNetCore](https://www.nuget.org/packages/Swashbuckle.AspNetCore) NuGet package to the ASP.NET Core web API project:
+* From the File menu, select New > Project.
+* Enter Web API in the search box.
+* Select the ASP.NET Core Web API template and select Next.
+* In the Configure your new project dialog, name the project WeatherAPI and select Next.
+* In the Additional information dialog:
+* Confirm the Framework is .NET 6.0 (Long-term support).
+* Confirm the checkbox for Use controllers (uncheck to use minimal APIs) is checked.
+* Confirm the checkbox for Enable OpenAPI support is checked.
+* Select Create.
 
-    ![Screenshot to configure the NuGet dialog](publish-to-azure-api-management-using-vs/_static/configure_nuget.png)
+## Explore the code
 
-1. Add the following line to the `Startup.ConfigureServices` method:
-    
-    ```csharp
-    services.AddSwaggerGen();
-    ```
-    
-1. Add the following line to the `Startup.Configure` method:
+Swagger definitions allow Azure API Management to read the app's API definitions. By checking the _Enable OpenAPI support_ checkbox during app creation, Visual Studio will automatically add the code to create the Swagger definitions. Open up the `Program.cs` file and you'll see the following code:
 
-    ```csharp
+```csharp
+
+...
+
+builder.Services.AddSwaggerGen();
+
+...
+
+if (app.Environment.IsDevelopment())
+{
     app.UseSwagger();
-    ```
+    app.UseSwaggerUI();
+}
+
+...
+
+```
+
+### Ensure the Swagger definitions are always generated
+
+Azure API Management needs the Swagger definitions to always be present, regardless of the application's environment. To ensure they are always generated, move `app.UseSwagger();` outside of the `if (app.Environment.IsDevelopment())` block. In addition, you can 
```

Comment:
> In addition, you can
can what?

BTW, in this repo we prefer to not use `you`. It's generally unnecessary.

Reply:
Updated in subsequent commit.